packages/bundle-analyzer/src/cli.ts

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+
+import path from "node:path";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { createAndUploadReport } from "./index.js";
+import { red, type Options } from "@codecov/bundler-plugin-core";
+
+interface Argv {
+  buildDirectoryPath: string;
+  dryRun: boolean;
+  uploadToken?: string;
+  apiUrl: string;
+  bundleName?: string;
+  debug: boolean;
+}
+
+const argv = yargs(hideBin(process.argv))
+  .usage("Usage: $0 <build-directory-path> [options]")
+  .command(
+    "$0 <build-directory-path>",
+    "Analyze and upload bundle report",
+    (yargs) => {
+      return yargs.positional("buildDirectoryPath", {
+        describe: "The path to the build directory",
+        type: "string",
+        demandOption: true,
+      });
+    },
+  )
+  .options({
+    "dry-run": {
+      alias: "d",
+      type: "boolean",
+      description: "Perform a dry run without uploading",
+      default: false,
+    },
+    "upload-token": {
+      alias: "t",
+      type: "string",
+      description: "Specify the upload token for authentication",
+    },
+    "api-url": {
+      alias: "a",
+      type: "string",
+      description: "Set the API URL",
+      default: "https://api.codecov.io",
+    },
```

Comment:
nit: Thoughts on switching `a` to `u`? To me there's just something odd with `a` being the URL we're hitting.

Reply:
sure! fixed!